src/transformers/pipelines/document_question_answering.py

```diff
@@ -129,6 +134,11 @@ class DocumentQuestionAnsweringPipeline(ChunkPipeline):
     [huggingface.co/models](https://huggingface.co/models?filter=document-question-answering).
     """
 
+    # Make sure the docstring is updated when the default generation config is changed
+    _default_generation_config = GenerationConfig(
+        max_new_tokens=256,
```

Comment:
```suggestion
        max_new_tokens=1024,
```

maybe a bit higher specifically if some one wants to just pass a long prompt

Reply:
it's for max new tokens (not total tokens) so it doesn't take into account the prompt! Same answer as below, I'd encourage not setting too high values either

Reply:
@LysandreJik @Vaibhavs10 

We might want to enable something like `max_new_length="auto"`, which adjusts `max_new_length` according to available HW? 

As VB wrote, in most models we don't actually hit `max_new_length` because of the EOS token(s). But a large default `max_new_tokens` will 100% hit OOM issues (it was one of the reasons behind the old default btw 👀 )

Reply:
yes, 100% as a follow-up